### ISSUE TITLE
Feature: Send File without preview

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1039]
Some people have been asking to send files (e.g images, mp3) without resulting in a PTT or an image type message etc.

**Comment by joedutra** [comment]
<img src="https://media.giphy.com/media/sNZddQMtrqpS0HEEMC/giphy.gif"></img>
Man, you know it sucks, how we copy your repo, if we have functions that you don’t even have, all for free, we perform almost every day, I came from Sulla and you too, but if you look at following the week, we redid it all lib again, now you will no longer be able to copy us, you copy and keep saying that we copy, nor compete with you ours and for free and your payment

**Comment by joedutra** [comment]
I come to apologize and say that we have nothing against this, only that they accuse us of copying and we never copied anything, if someone copied it was when someone changed our lib, I apologize and the intention was not to be slandered anymore of copying something we worked on every day. just like you, we will not do that anymore.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.2](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.1...3.1.2) (2020-12-02)

- Release 3.1.2 [`297495d`](https://github.com/open-wa/wa-automate-nodejs/commit/297495da4eb2258f3d43c72fd3b90914717cbfa4)
- Feature: Now able to send file without a preview #1039 [`9ed6c43`](https://github.com/open-wa/wa-automate-nodejs/commit/9ed6c4385f46dae17152479da8c49fbcc0546fc2)
- dev: implemented documentation cname fixer [`6e5cd10`](https://github.com/open-wa/wa-automate-nodejs/commit/6e5cd10ec856c59f79c8a6eb22bb2e597181e9de)
- Create CNAME [`ee7b797`](https://github.com/open-wa/wa-automate-nodejs/commit/ee7b7978b4194da45e9b58141d16febfe0ee7467)

**Comment by smashah** [comment]
@zalviandyr @gimenz You can now send a file without a preview/as a downloadable file by setting the `withoutPreview` parameter to true

![image](https://user-images.githubusercontent.com/5903821/100823993-3ae2cb80-344d-11eb-916a-5f79fbb394be.png)
